pkgs/data/misc/cacert/default.nix

```diff
@@ -66,6 +70,52 @@ stdenv.mkDerivation {
   setupHook = ./setup-hook.sh;
 
   passthru.updateScript = ./update.sh;
+  passthru.tests = {
+    # Test that building this derivation with a blacklist works, and that UTF-8 is supported.
+    blacklist-utf8 = let
+      blacklistCAToFingerprint = {
+        # "blacklist" uses the CA name from the NSS bundle, but we check for presence using the SHA256 fingerprint.
+        "CFCA EV ROOT" = "5C:C3:D7:8E:4E:1D:5E:45:54:7A:04:E6:87:3E:64:F9:0C:F9:53:6D:1C:CC:2E:F8:00:F3:55:C4:C5:FD:70:FD";
+        "NetLock Arany (Class Gold) Főtanúsítvány" = "6C:61:DA:C3:A2:DE:F0:31:50:6B:E0:36:D2:A6:FE:40:19:94:FB:D1:3D:F9:C8:D4:66:59:92:74:C4:46:EC:98";
+      };
+      mapBlacklist = f: concatStringsSep "\n" (mapAttrsToList f blacklistCAToFingerprint);
+    in runCommand "verify-the-cacert-filter-output" {
+      inherit openssl;
```

Comment:
add `openssl` to `nativeBuildInputs` as that will probably (hopefully?) allow this to succeed even in a cross compiled build. You can then just refer to `openssl` verbatim instead of writing `$openssl/bin/openssl`.

Reply:
Done. Tested with `nix-build . -A pkgsCross.raspberryPi.cacert.tests`, produced "/nix/store/z0rxlz0m6zxakkhby24h0hf0jh9v0qfh-verify-the-cacert-filter-output"